.renovaterc.json5

```
@@ -124,7 +124,9 @@
       ],
       "matchStrings": [
         "\"openApiSupportedVersion\": \"~(?<currentValue>[^\"]+)\""
-      ]
+      ],
+      // TODO wait for https://github.com/OpenAPITools/openapi-generator/issues/20135 to be fixed before upgrading to 7.10.0
+      "enabled": false
```

Maybe it would help to get notified when fixed with:

```suggestion
      "allowedVersions": "!/7\\.10\\.0$/"
```

Otherwise we may forgot to re-enable it :S

---

As discussed, please refer the issue relative to the test of the generator as reply of this comment (to keep PR link).

---

As discussed will be re-enabled once https://github.com/AmadeusITGroup/otter/issues/2674 is done ! :)